earth_enterprise/src/fusion/gepublish/CurlRequest.cpp

```diff
@@ -193,14 +199,17 @@ bool UploadRequest::Start(bool report_progress, geProgress* progress) {
       return false;
     }
     Close();
+    processed_size += entry.data_size;
```

Comment:
I moved it here because when I added the else clause to the if statement the code was redundant in both blocks of code.  So if I was going to the do the calculation no matter what I might as well always do it outside the if clause.  Should yield the same result because before the change we only did the calculation and traced the result if report_progress and progress where true.  And that would not change for the duration of the function so it always did it for each loop iteration or it never did it.  I changed things to always calculate and trace the result depending on trace level set.